### ISSUE TITLE
Choppy text.

### ISSUE THREAD

**Issue by Aegarain** [Deli-Collective/DeliCounter#12]
I think the text is either being rendered at a low resolution, or it doesn't have any aliasing, but regardless it isn't nice to read

**Comment by nrgill28** [comment]
The text looks fine to me. Could you elaborate / screenshot where it's showing up like this?

**Comment by Aegarain** [comment]
![JaggedText](https://user-images.githubusercontent.com/63934486/111881852-bcfcee80-896f-11eb-81dc-22d15a74b4a5.png) It's easy to see if you look at diagonal lines, like the capital A. I included the H3VR shortcut for reference. It looks better online due to the image interpolation, its more pronounced on my monitor

**Comment by nrgill28** [comment]
This is what I see on v0.1.12.
![image](https://user-images.githubusercontent.com/22647801/111882017-9946a200-8989-11eb-94b7-80af2310f438.png)

I'm not entirely sure what would be causing the text to display as it does for you, although the problem seems closer to a scaling issue than an anti-aliasing issue (since that text in your screenshot is clearly anti-aliased). I think you previously mentioned that you were running a 1440p monitor, do you happen to also have your Windows scaling setting set to something higher than 100%?

**Comment by Aegarain** [comment]
I'll be able to check in a bit

**Comment by Aegarain** [comment]
No, my scaling is set to 100%

**Comment by nrgill28** [comment]
Hm, interesting. Is this the only program you're noticing this with? Since I can't reproduce the issue and I've not had anyone else report it either I'm not convinced it'd be an issue with Deli Counter

**Comment by Aegarain** [comment]
Yeah, it's just deli counter. I have no idea what would be causing it

**Comment by malicean** [comment]
I also have a 1440p monitor. I thought it looked a bit chunky the first time I opened it, but dismissed it. I can only tell in the mod listings, not the additional info.

![image](https://user-images.githubusercontent.com/5615358/111890229-83e76d00-89b5-11eb-9bca-2e9deabe901c.png)

**Comment by Maiq-The-Dude** [comment]
I primarily use a 4k monitor with windows scaling at 100% and have issues with text clarity as well, just to add on.

**Comment by nrgill28** [comment]
Alright so that's super weird then. I'm using the default font and styling from a UI framework so 🤷🏻‍♂️. I'll poke around some to see if others using the same library have similar issues but otherwise I'm not quite sure where else to look
